scripts/lib/CIME/tests/case_fake.py

```diff
@@ -97,9 +98,11 @@ def create_clone(self, newcase, keepexe=False, mach_dir=None, project=None,
         os.makedirs(newcaseroot)
         clone = self.copy(newcasename = newcasename, newcaseroot = newcaseroot)
         if cime_output_root is not None:
-            self.set_value('CIME_OUTPUT_ROOT', cime_output_root)
+            clone.set_value('CIME_OUTPUT_ROOT', cime_output_root)
+        if exeroot is not None:
+            clone.set_value('EXEROOT', exeroot)
         if rundir is not None:
-            self.set_value('RUNDIR', rundir)
+            clone.set_value('RUNDIR', rundir)
```

Comment:
Change from self.set_value to clone.set_value was a mostly unrelated bug fix that just affects unit tests (and not in any substantive way).